actions/rules/action_state_to_metastate.go

```diff
@@ -177,6 +178,9 @@ func (act ActionStateToMetaState) getStateToMetastateMap(workitemTypeID uuid.UUI
 	if err != nil {
 		return nil, err
 	}
+  if len(stateList) != len(metastateList) {
+		return nil, errs.New("inconsistent number of states and metatstates in the current template")
```

Comment:
Please use `errs.Wrapf` and give the numbers of each states. Also, should it maybe say *[...] in the current work item type*?

Reply:
Fixed.